pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -727,19 +727,23 @@ def _handle_distribution_tree(declarative_content):
             if variants:
                 Variant.objects.bulk_create(variants)
 
-        update_references_to_save = []
-        update_collection_packages_to_save = []
-        modulemd_pkgs = []
-        group_pkgs = []
-        category_groups = []
-        env_groups = []
-        env_optgroups = []
+        UpdateRecordCollections = UpdateRecord.collections.through
         ModulemdPackages = Modulemd.packages.through
         PackageGroup_packages = PackageGroup.related_packages.through
         PackageCategory_groups = PackageCategory.packagegroups.through
         PackageEnvironment_groups = PackageEnvironment.packagegroups.through
         PackageEnvironment_optionalgroups = PackageEnvironment.optionalgroups.through
```

Comment:
I think we are inconsistent here with the naming. Some relations are named using camel case only, some are combination camel and snake ones. Do you think it's worth making it consistent?